opennms-doc/guide-install/src/asciidoc/text/minion/introduction.adoc

```diff
@@ -2,16 +2,69 @@
+The network area where access to managed network devices and applications is allowed can be modeled in a _Location_.
+Monitored _Nodes_ and _IP Services_ are associated to _Locations_ and are defined during _Provisioning_.
+Each _Minion_ is configured with a _Location_ and all _Nodes_ and _IP Services_ in the same _Location_ are monitored through this _Minion_.
 
-NOTE: Before attempting to setup _Minion_, you must have an instance of _OpenNMS_ set up using the same version of the packages.
+NOTE: The _Minion_ is currently not designed to be a replacement for the _Remote Poller_.
+      With _Remote Poller_ a service is additionally tested from multiple remote sites, whereas a _Minion_ extends network reachability for a central _{opennms-product-name}_ instance.
 
-_Minion_ can be installed on the same system as _OpenNMS_ or on other systems provided that it can communicate with:
+[[gi-install-minion-location]]
+._Nodes_ with _Minions_ in _Locations_
+image::minion/01_minion-location.png[]
 
-. The OpenNMS REST interface
-. The ActiveMQ broker used by OpenNMS
+In figure <<gi-install-minion-location, _Nodes_ with _Minions_ in _Locations_>> an example with isolated branch offices is shown.
+Every _Node_ created in _{opennms-product-name}_ is by default created in the location named _Default_.
+All _Nodes_ and _Services_ in _Default_ location are handled by the central _{opennms-product-name}_ instance itself.
+For each branch office in an isolated network, a _Location_ is defined.
+The _Minion_ has a configuration property for the _Location_ and will register itself on first startup on the _{opennms-product-name}_ instance.
+
+The _Provisioning System_ allows to associate _Nodes_ to a _Location_.
+_{opennms-product-name}_ will delegate monitoring requests for _Nodes_ in the specified _Locations_ to the registered _Minions_ and uses them as a proxy.
+
+Figure <<gi-install-minion-communication, Minion communication>> gives a more detailed overview about the communication between an _{opennms-product-name}_ instance and a _Minion_.
+
+[[gi-install-minion-scenario]]
+.Minion communication
+image::minion/02_minion-communication.png[]
+
+The _Minion_ needs a configuration which requires at minimum the following information:
+
+* An unique identifier for this specific _Minion_ as `id`
+* Monitoring location as a name for a network area this _Minion_ is responsible for nodes named as `location`
+* The communication endpoints for the central _{opennms-product-name}_ instance as `broker-url` and `http-url`
+
+The configuration is in a property file in `${MINION_HOME}/etc/org.opennms.minion.controller.cfg`.
+When the minimal configuration is set up the _Minion_ can be started and initially connects to the central _{opennms-product-name}_ instance and identifies itself with his unique _ID_.
+
+NOTE: If the _ID_ for a _Minion_ is not configured manually, the first start generates a unique _ID_ using `/usr/bin/uuidgen -t`.
```

Comment:
This ID is actually generated when the packages are installed. On upgrade, it won't update the ID if it's not the default one though.